frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -51,6 +51,22 @@ pods:
           {{/TLS_ENABLED}}
         env:
           JOURNALNODE: true
+        # The existence of the journal-data/hdfs/current/committed-txid file indicates HDFS has already been deployed once
```

Comment:
At the surface it's obvious what it's doing, but yes it'd be helpful if it described _why_ it was doing that specifically.